indico/modules/categories/forms.py

```diff
@@ -0,0 +1,89 @@
+
+    title = StringField(_("Title"), [DataRequired()])
+    description = IndicoMarkdownField(_("Description"),
+                                      description=_("You can use Markdown or basic HTML formatting tags."))
+    timezone = IndicoTimezoneSelectField(_("Timezone"), [DataRequired()],
+                                         description=_("Default timezone event lists will show up in. It will also be "
+                                                       "used as a default for new events."))
+    lecture_theme = IndicoThemeSelectField('simple_event', _("Theme for Lectures"), [DataRequired()],
+                                           description=_("Default timetable theme used for lecture events"))
+    meeting_theme = IndicoThemeSelectField('meeting', _("Theme for Meetings"), [DataRequired()],
+                                           description=_("Default timetable theme used for meeting events"))
+    visibility = SelectField(_("Event visibility"), [Optional()], coerce=lambda x: None if x == '' else int(x),
+                             description=_("""From which point in the category tree contents will be visible from """
+                                           """(number of categories upwards). Applies to "Today's events" and """
+                                           """Calendar. If the category is moved, this number will be preserved."""))
+    suggestions_disabled = BooleanField(_('Disable Suggestions'), widget=SwitchWidget(),
+                                        description=_("Enable this if you don't want Indico to suggest this category as"
+                                                      " a possible addition to a user's favourites."))
+    event_message_mode = IndicoEnumSelectField(_("Message Type"), enum=EventMessageMode,
+                                               default=EventMessageMode.disabled,
+                                               description=_("This message will show up at the top of every event page "
+                                                             "in this category"))
+    event_message = IndicoMarkdownField(_("Content"),
+                                        description=_("You can use Markdown or basic HTML formatting tags."))
+    event_creation_notification_emails = EmailListField(_("Notification E-mails"),
+                                                        description=_("List of e-mails that will receive a notification"
+                                                                      " every time a new event is created. One e-mail "
+                                                                      "per line."))
+
+    def __init__(self, *args, **kwargs):
+        super(CategorySettingsForm, self).__init__(*args, **kwargs)
+        category = kwargs.pop('category')
+        self.visibility.choices = calculate_visibility_options(category)
+        self.visibility.default = ((category.parent.visibility + 1) if
```

Comment:
I don't think the default value is ever used - we don't have it in the creation form and when editing there's always a value from the DB